food-truck-frontend/src/components/notifications/NotificationD2P.ts

```diff
@@ -24,11 +22,7 @@ async function updateNotifications(
 
 const mapDispatchToProps = (dispatch: Dispatch<NotificationAction>) => {
   return {
-    loadNotificationsFromBackend: () => {
-      return new Promise<void>(() => {
-        updateNotifications(dispatch);
-      });
-    },
+    loadNotificationsFromBackend: () => updateNotifications(dispatch),
```

Comment:
Is update notifications now a synchronous method? If not, why did we change it from being a Thunk to a standard dispatch?

Reply:
updateNotifications already returned a Promise<void> so we can just return that instead of wrapping it in a new promise

Reply:
Works for me. Thanks.